app/assets/javascripts/Components/markus_file_manager.jsx

```diff
@@ -298,7 +298,7 @@ class FileManagerFile extends FileRenderers.RawTableFile {
     } else {
       icon = <i className="fa fa-file-o" aria-hidden="true" />;
     }
-
+    let download_icon = <i className="fa fa-download-o" aria-hidden="true" />;
```

Comment:
Add a `title` attribute here so that if a user hovers over it, they'll know what it does.